### ISSUE TITLE
Talk about Learning Rate scheduler in paper

### ISSUE THREAD

**Issue by amghr** [csml-beach/deep-bayesian-operator-nets#27]
we have implemented lr to reduce the sensibility to hyper parameter and to also navigate local minima better

**Comment by amghr** [comment]
added this page 18 `We have implemented a learning rate scheduler that adaptively lowers the learning rate to further reduce the sensitivity to hyperparameters and help the model navigate local minima more effectively`
